TicketingSystem.ApiService/BackgroundWorkers/SeatReleasingService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using TicketingSystem.ApiService.Services.PaymentService;
+
+namespace TicketingSystem.ApiService.BackgroundWorkers
+{
+    public class SeatReleasingService : BackgroundService
+    {
+        private readonly IServiceScope _scope;
+        private readonly TimeSpan CheckTime = TimeSpan.FromMinutes(1);
+        public SeatReleasingService(IServiceProvider serviceProvider)
+        {
+            _scope = serviceProvider.CreateScope();
+        }
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+            var paymentService = _scope.ServiceProvider.GetRequiredService<IPaymentService>();
+            while (!stoppingToken.IsCancellationRequested)
+            {
+                await Task.Delay(CheckTime, stoppingToken);
+                await paymentService.FailOutdatedPayments();
```

Comment:
In general, for long running jobs it might be a good idea to handle exceptions that may occur in the calling methods. Also, note that when `stoppingToken` cancellation is requested, `Task.Delay` will throw task cancellation exception. This is not really significant for a background service since `stoppingToken` cancellation is only requested during shutdown, but you should always keep in mind task cancellation exception may occur in any method accepting `CancellationToken` is passed. However, in most cases it is propagated and handled by some middleware